wire-library/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/internal/parser/FieldElement.kt

```diff
@@ -55,9 +54,9 @@ data class FieldElement(
    * Both `default` and `json_name` are defined in the schema like options but they are actually
    * not options themselves as they're missing from `google.protobuf.FieldOptions`.
    */
-  private fun optionsWithSpecialValues(syntaxRules: SyntaxRules): List<OptionElement> {
+  private fun optionsWithSpecialValues(): List<OptionElement> {
     var options =
-        if (defaultValue == null || !syntaxRules.allowUserDefinedDefaultValue()) {
```

Comment:
Doesn't make sense to not parse it. The parsing shouldn't know about that and the linker, who does, will add an error for us. I so removed all syntax rules in the `toSchema` because not needed anymore.